cpp/include/tensorrt_llm/common/config.h

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2022-2025, NVIDIA CORPORATION.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+#ifndef TRTLLM_CONFIG_H
+#define TRTLLM_CONFIG_H
+
+#if (defined(TRTLLM_NAMESPACE_PREFIX) || defined(TRTLLM_NAMESPACE_POSTFIX)) && !defined(TRTLLM_NAMESPACE_QUALIFIER)
+#error TRTLLM requires a definition of TRTLLM_NAMESPACE_QUALIFIER when TRTLLM_NAMESPACE_PREFIX/POSTFIX are defined.
+#endif
+
+/**
+ * \def TRTLLM_WRAPPED_NAMESPACE
+ * If defined, this value will be used as the name of a namespace that wraps the
+ * `tensorrt_llm::` namespace.
+ * If THRUST_TRTLLM_WRAPPED_NAMESPACE is set, this will inherit that macro's value.
```

Comment:
Cleanup all references to `thrust` and `cub` in this file.

Reply:
Remove unnecessary macros and clean the references.